CalorieTracker/CalorieTableViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  CalorieTableViewController.swift
+//  CalorieTracker
+//
+//  Created by Lambda_School_Loaner_201 on 2/22/20.
+//  Copyright © 2020 Christian Lorenzo. All rights reserved.
+//
+
+import UIKit
+import CoreData
+import SwiftChart
+
+class CalorieTableViewController: UITableViewController, NSFetchedResultsControllerDelegate, ChartDelegate {
+    
+    override func viewDidLoad() {
+            super.viewDidLoad()
+            barButtonItems()
+        NotificationCenter.default.addObserver(self, selector: #selector(refresh), name: .caloriesAded, object: nil)
+        }
+        
+        override func viewWillAppear(_ animated: Bool) {
+            super.viewWillAppear(animated)
+            calorieTrackerController.fetchTracks()
+            addChartData()
+
+            
+        }
+        
+
+        override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+            return calorieTrackerController.trackedCalories.count
+        }
+        
+        
+        override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: "CalorieTableViewCell",
+                                                           for: indexPath) as? CalorieTableViewCell else { return UITableViewCell() }
+            let tracked = calorieTrackerController.trackedCalories[indexPath.row]
+            
+            cell.textLabel?.text = "Calories: \(tracked.caloriesCount!)" //Calories Cell.
+                
+            let formated = DateFormatter()
+            formated.dateStyle = .medium
+            formated.timeStyle = .short
+            let dateString = formated.string(from: tracked.date!)
+            cell.detailTextLabel?.text = dateString // Date cell.
+     
+            return cell
+        }
+        
+        private func barButtonItems() {
+            navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addCalorie))
+            
+            // MARK: Remove only for testing
```

Comment:
Commenting code is an excellent way to communicate to other developers about what's going. on.